src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -65,6 +70,12 @@ extern "C" {
 #define MAX_STATS_STRING_LENGTH 255U
 /*!@} */
 
+/**
+ * Maximum length of candidate type (host, srflx, relay, prflx, unknown)
+ */
+#define MAX_CANDIDATE_TYPE_LENGTH 10U
```

Comment:
I also had this question. I assumed it was incase there are some longer, unexpected values that it could possibly be, but I see this char array is only ever set via one of the values from our enum:
```
typedef enum {
    ICE_CANDIDATE_TYPE_HOST = 0,             //!< ICE_CANDIDATE_TYPE_HOST
    ICE_CANDIDATE_TYPE_PEER_REFLEXIVE = 1,   //!< ICE_CANDIDATE_TYPE_PEER_REFLEXIVE
    ICE_CANDIDATE_TYPE_SERVER_REFLEXIVE = 2, //!< ICE_CANDIDATE_TYPE_SERVER_REFLEXIVE
    ICE_CANDIDATE_TYPE_RELAYED = 3,          //!< ICE_CANDIDATE_TYPE_RELAYED
} ICE_CANDIDATE_TYPE;
```
Which correspond to:
```
#define SDP_CANDIDATE_TYPE_HOST    "host"
#define SDP_CANDIDATE_TYPE_SERFLX  "srflx"
#define SDP_CANDIDATE_TYPE_PRFLX   "prflx"
#define SDP_CANDIDATE_TYPE_RELAY   "relay"
#define SDP_CANDIDATE_TYPE_UNKNOWN "unknown"
```

Since "unknown" has a length of 7 and we already add 1 to it for null terminator when defining the arrays, I will set `MAX_CANDIDATE_TYPE_LENGTH` to 7.

Reply:
Yep, those are the only types. 

Unknown isn't even in the spec: https://datatracker.ietf.org/doc/html/rfc5245#section-15.1

```
<cand-type>:  encodes the type of candidate.  This specification
      defines the values "host", "srflx", "prflx", and "relay" for host,
      server reflexive, peer reflexive, and relayed candidates,
      respectively.  The set of candidate types is extensible for the
      future.
```